r/tools/autobrew

```diff
@@ -48,7 +48,8 @@ fi
 # Hardcode this for my custom autobrew build
 rm -f $BREWDIR/lib/*.dylib
 AWS_LIBS="-laws-cpp-sdk-config -laws-cpp-sdk-transfer -laws-cpp-sdk-identity-management -laws-cpp-sdk-cognito-identity -laws-cpp-sdk-sts -laws-cpp-sdk-s3 -laws-cpp-sdk-core -laws-c-event-stream -laws-checksums -laws-c-common -lpthread -lcurl"
-PKG_LIBS="-L$BREWDIR/lib -lparquet -larrow_dataset -larrow -larrow_bundled_dependencies -lthrift -llz4 -lsnappy -lzstd $AWS_LIBS"
+PKG_LIBS="-lparquet -larrow_dataset -larrow -larrow_bundled_dependencies -lthrift -llz4 -lsnappy -lzstd $AWS_LIBS"
+PKG_DIRS="-L$BREWDIR/lib"
```

Comment:
We will need to remember to upstream this change when we release to CRAN

Reply:
What is the relationship of this script (`r/tools/autobrew`) to https://github.com/autobrew/scripts/blob/master/apache-arrow?

Reply:
It's based on an earlier version of Jeroen's autobrew, and it's closest now to how he actually builds the C++ dependencies. It looks like the new autobrew script is based on some artifacts that are bundled up in a separate process (see https://github.com/autobrew/bundler/blob/master/apache-arrow.sh).

Reply:
I created ARROW-11861 to remind us of this.

Reply:
Turns out we also needed to add  `-DARROW_R_WITH_PARQUET` and `-DARROW_R_WITH_DATASET` to the cflags in ttps://github.com/autobrew/scripts/blob/master/apache-arrow (ARROW-12604)